force-app/main/default/classes/AttendanceService.cls

```diff
@@ -12,8 +12,16 @@ public with sharing class AttendanceService {
     private AttendanceSelector attendanceSelector = new AttendanceSelector();
     @TestVisible
     private ServiceDeliveryDomain domain = new ServiceDeliveryDomain();
+    @TestVisible
+    private AttendanceSystemAccess systemAccess = new AttendanceSystemAccess();
 
     private static final String DEFAULT_STATUS = 'Present';
```

Comment:
I have one question here. It is really a dumb one. We are setting the default status to `Present`. What if the user calls `Present` something else then how would it work. I may be missing something here.